mailscanner/lib/lineConfig.js

```diff
@@ -94,6 +94,7 @@ function printLineGraph(chartId, settings) {
               showLine: (typeof settings.types === 'undefined' || settings.fillBelowLine[i] ? true :
                           (typeof settings.drawLines === 'undefined' ? false : settings.drawLines)),
               pointRadius: 1,
+              borderWidth: 0.1,
```

Comment:
Unfortunately, it's not good.

![capture d ecran 2017-07-03 a 11 53 38](https://user-images.githubusercontent.com/12907102/27787811-73640972-5fe6-11e7-96ee-14689f212d2d.png)

The problem should come from another place as with PT1H we don't have the problem.

![capture d ecran 2017-06-29 a 19 07 34](https://user-images.githubusercontent.com/12907102/27700606-691cf5fe-5cfe-11e7-951f-84624c35eddf.png)